src/main/java/com/linkedin/kmf/services/TopicManagementService.java

```diff
@@ -23,10 +23,16 @@
  */
 public class TopicManagementService implements Service {
   private final MultiClusterTopicManagementService _multiClusterTopicManagementService;
+  private static final CompletableFuture<Void> TOPIC_PARTITION_READY = new CompletableFuture<>();
```

Comment:
What  happens if there is more than one topic management service in a process?

Reply:
updated.